tools/merge_ratbagd.py

```diff
@@ -30,33 +30,35 @@
 def write_ratbagctl(
     output_file: TextIO, ratbagctl_path: str, ratbagd_path: str, version_string: str
 ) -> None:
-    with open(ratbagctl_path, 'r', encoding='utf-8') as ratbagctl, open(ratbagd_path, 'r', encoding='utf-8') as ratbagd:
+    with open(ratbagctl_path, encoding="utf-8") as ratbagctl, open(
```

Comment:
For reference, if we were on Python >=3.9, you could have the objects for the context manager inside parenthesis, making this look better.

```python
with (
    open(...) as file1,
    open(...) as file2,
):
    ...
```